index.html

```diff
@@ -246,9 +252,41 @@ <h4>
         <button class="container-btn" type="button" name="button">
           See Project
         </button>
-      </section>
+      </section> -->
     </section>
 
+  <!-- <section id="mobile-pop-up" class="mobile-pop-up-invisible">
+
+  </section> -->
+
+    <!-- <section id="mobile-pop-up">
+      <div class="mobile-pop-up-container-image">
+        <img class="mobile-pop-up-item-image" src="${item.image}" alt="ueab-university" />
+      </div>
+      <div class="mobile-pop-up-container-title">
+        <h2>Keeping track of hundreds of components</h2>
+      </div>
+      <div class="mobile-pop-up-container-projects">
+        <ul class="container-projects-element">
+          <li>Trial A</li>
+          <li>Trial B</li>
+          <li>Trial C</li>
+          <li>Trial D</li>
+        </ul>
+      </div>
+      <div class="mobile-pop-up-text">
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+      </div>
+      <div class="mobile-button-pop-up">
+        <button class="mobile-pop-up-container-btn" type="button" name="button">
+          See Live
+        <i class="fab fa-github fa-sm social"></i></button>
+        <button class="mobile-pop-up-container-btn" type="button" name="button">
+          See Source
+        <i class="fab fa-github fa-sm social"></i></button>
+      </div>
+    </section> -->
+
```

Comment:
- [ ] **[OPTIONAL]** Let's keep our code clean by removing all the commented lines of code in this project as I can see so many commented code. This will help us to achieve a clean codebase.